docs/editors-guide/project-management.md

```diff
@@ -19,59 +19,23 @@ Role | Description | Person(s) | Relevant issues
-
-## Liaise with Mondo Users
-
-- Keep track of Mondo users (keep contact info up to date, what project they are working on, how they are using Mondo)
-
-## Onboarding of new contributors
-
-- Refer to the [onboarding doc here](https://docs.google.com/spreadsheets/d/18qfgcUYaCif0NhfNF5QaLp8DvxG0llwcSmp3B5U-MhE/edit#gid=0)
+## Mondo tasks
+
+Task | Description | Who
+-- | -- | --
+Manage meetings | Mondo meetings should be added to the Monarch shared google calendar. Members of the curation and technical teams are outlined [here](https://mondo.monarchinitiative.org/pages/contributors/). | Project Manager
+Mondo workshops | Any upcoming workshops should be organized and advertised. Ideally we should have community based Mondo workshops every 4-6 months.| Project Manager
+Advertising workshops | Workshops should be advertised on our [website](https://mondo.monarchinitiative.org/pages/workshop/), on our Slack channel, on the mailing list | Project Manager
+Set milestones and triage tickets |All [tickets](https://github.com/monarch-initiative/mondo/issues) should be: 1. Acknowledged, thank the contributor for the ticket, 2. Assigned to a curation or technical team member, 3. Assigned to a monthly milestone, 4. Given a label | Lead Curator
+Mondo Releases |Mondo should be released on a monthly basis on/around the first of the month. Please refer to the [pre-release checklist](https://mondo.readthedocs.io/en/latest/editors-guide/pre-release-checklist/) for necessary tasks before the release. Instructions are [here](https://mondo.readthedocs.io/en/latest/developer-guide/release/). If there are any delays with the release (more than a couple days), please communicate these to the Mondo users on the mailing list and in Slack. Some of our users depend on the regular releases, such as EFO. | Curation team
+Grant reporting | 1. Link Mondo tickets to [Phenomics First Operations Board](https://github.com/monarch-initiative/phenomics_first_resource/issues?q=is%3Aissue+is%3Aopen+label%3A%22Aim+2+-+MONDO%22) 2. Ensure deliverables for P1 grant are being addressed, 3. Send all dissemination artifacts to the 'nuggets' address for Monarch/TISLab (to help RPPR prepping) | Project Manager
+RPPR | Coordinate reporting for yearly RPPR | Project Manager
+Website updates |The [Mondo website](https://mondo.monarchinitiative.org/) should be regularly updated (instructions are [here](https://mondo.readthedocs.io/en/latest/editors-guide/mondo-website-editing/)). Regular updates that may be needed include: [Mondo users](https://mondo.monarchinitiative.org/pages/users/), [Mondo contributors](https://mondo.monarchinitiative.org/pages/contributors/), [Resources](https://mondo.monarchinitiative.org/pages/resources/)- including new papers, presentations, blog posts, etc., [Workhops](https://mondo.monarchinitiative.org/pages/workshop/), [FAQs](https://mondo.monarchinitiative.org/pages/faq/) should be updated as new questions arise | Curation team + Project Manager
```

Comment:
Same here, I would only add who is _responsible_ for making the edits, not who makes the edits.